### PR TITLE
Fix e2e-aws-csi test jobs

### PR DESCRIPTION
**Pull request by jsafrane**
Fix openshift-e2e-aws-csi workflow not to install CSI driver (because it's already installed by cluster setup) and to use CSI driver manifest from aws-ebs-csi-driver-operator-test image (created during the operator build).

Expected `e2e-aws-csi-workflow` workflow:
* Install the cluster as usual (`ipi-aws-pre` chain)
* Copy CSI test manifest from the last AWS EBS CSI operator image to $SHARED_DIR (`storage-conf-csi-aws-ebs` step)
* Run e2e tests with `TEST_CSI_DRIVER_MANIFEST=<the manifest>` (`openshift-e2e-test`)
  * `openshift-e2e-test` uses `$TEST_CSI_DRIVER_MANIFEST` as `TEST_CSI_DRIVER_FILES` and runs openshift/csi tests

WIP:
  * ~extra debugging output in some scripts~

### CODE REVIEW

File: ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh [hunk 1]
```diff
@@ -10,6 +10,10 @@ export GCP_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/gce.json
 export HOME=/tmp/home
 export PATH=/usr/libexec/origin:$PATH
 
+if [[ -n "${TEST_CSI_DRIVER_MANIFEST}" ]]; then
+    export TEST_CSI_DRIVER_FILES=${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST}
+fi
```
Comment (bertinatto) [inline ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh:15]: This just exports TEST_CSI_DRIVER_FILES that's used by openshift-tests

File: ci-operator/step-registry/openshift/e2e/aws/csi/openshift-e2e-aws-csi-workflow.yaml [hunk 2]
```diff
@@ -3,12 +3,13 @@ workflow:
...
     pre:
     - chain: ipi-aws-pre
+    - ref: storage-conf-csi-aws-ebs
     test:
     - ref: openshift-e2e-test
     post:
     - chain: ipi-aws-post
     env:
-      TEST_INSTALL_CSI_DRIVERS: aws-ebs
+      TEST_CSI_DRIVER_MANIFEST: manifest-aws-ebs.yaml
```
Comment (bertinatto) [inline ci-operator/step-registry/openshift/e2e/aws/csi/openshift-e2e-aws-csi-workflow.yaml:12]: Changes the current workflow to NOT install CSI driver using openshift-tests.

Comment (bertinatto) [reply ci-operator/step-registry/openshift/e2e/aws/csi/openshift-e2e-aws-csi-workflow.yaml:12]: nobody reads TEST_CSI_DRIVER_MANIFEST so far

Comment (bertinatto) [reply ci-operator/step-registry/openshift/e2e/aws/csi/openshift-e2e-aws-csi-workflow.yaml:12]: TEST_CSI_DRIVER_MANIFEST will be set below, and openshift-tests will use it

**Comment by jsafrane** [comment]
I checked all jobs that fail all rehearsals and none of them seems to be caused by this change. Mostly they fail during cluster installation.

**Review by petr-muller — APPROVED** [review]

**Pull request merged**